style.css

```diff
@@ -3,34 +3,58 @@
   font-family: sans-serif;
   letter-spacing: 2px;
 }
+
 html {
   font-size: 15px;
 }
+
 header {
   padding: 10px;
   display: flex;
   justify-content: center;
 }
+
 body {
   background-color: #568203;
 }
+
+.menu-screen-container {
```

Comment:
the content in this isnt centered, looks weird

Reply:
Yep, we will be tweaking shortly

Reply:
tweaks made